atlasdb-commons/src/main/java/com/palantir/util/AssertUtils.java

```diff
@@ -119,7 +121,10 @@ public static void assertAndLogWithException(boolean cheapTest, String msg, Thro
     public static void assertAndLogWithException(Logger log, boolean cheapTest, String format, Throwable t,
             Object... args) {
         if (!cheapTest) {
-            assertAndLogWithException(log, false, String.format(format, args), t);
+            Object[] newArgs = Arrays.copyOf(args, args.length + 1);
+            newArgs[args.length] = t;
+            log.error("Assertion " + format + " with exception ", newArgs);
```

Comment:
Unfortunately this generally isn't allowed in gradle-baseline. I'm not sure what a good solution here is: separately doing a string substitution, maybe?